site/en/docs/extensions/mv3/devtools/index.md

```diff
@@ -378,8 +378,8 @@ window.addEventListener('message', function(event) {
   var message = event.data;
 
   // Only accept messages that we know are ours
-  if (typeof message !== 'object' || message === null ||
-      !message.source === 'my-devtools-extension') {
+  if (message !== null && typeof message !== 'object' ||
+       message.source !== 'my-devtools-extension') {
```

Comment:
Thanks for catching the bug in the second line of this if statement. I'd like to see one minor change: we should remove the extra space to keep the indentation at 6 spaces.

```suggestion
      message.source !== 'my-devtools-extension') {
```